kratos/tests/cpp_tests/utilities/test_debug_helpers.cpp

```diff
@@ -0,0 +1,35 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+//
+
+// Project includes
+#include "includes/debug_helpers.h"
+#include "includes/ublas_interface.h"
+#include "testing/testing.h"
+
+namespace Kratos
+{
+namespace Testing
+{
+KRATOS_TEST_CASE_IN_SUITE(DebugHelpersKRATOS_WATCH_VECTOR_WITH_PRECISION, KratosCoreFastSuite)
+{
+    Vector temp(1, 10.0);
+    KRATOS_WATCH_VECTOR_WITH_PRECISION(temp, 12);
```

Comment:
can you add a comment as to why you are adding the "test"?

Reply:
done :)